src/DragState.js

```diff
@@ -24,7 +57,8 @@ class DragState extends React.Component {
 }
 
 DragState.propTypes = {
-  children: PropTypes.func.isRequired
+  children: PropTypes.func.isRequired,
+  subscribeTo: PropTypes.array
```

Comment:
Isn't it required union of array and null?

Reply:
If it has a default does it need to be marked as required? I'm not super familiar with propTypes.

Reply:
I think type is checked after applying defaults. If this code is run without errors then prop types probably consider nullable === not required which is bad.

Reply:
Does Proptypes actually have a type for null? The documentation doesn't say anything about it. Maybe I'm not understanding correctly. The desired behavior would be for Prop types to only throw an error if the `subscribeTo` prop is present and has a value other than an array. Isn't that what it's doing now?

Reply:
I've done some more testing and as far as I can tell the Proptypes are behaving as expected. I'm going merge this in for now, but let me know if you find a scenario where it doesn't work as expected.